Index: tex-buf.el
===================================================================
RCS file: /cvsroot/auctex/auctex/tex-buf.el,v
retrieving revision 1.181
retrieving revision 1.183
diff -c -r1.181 -r1.183
*** tex-buf.el	25 Nov 2002 14:42:35 -0000	1.181
--- tex-buf.el	15 Dec 2002 02:15:34 -0000	1.183
***************
*** 962,967 ****
--- 962,969 ----
  			      (re-search-backward "[\r\n]" nil t)
  			      (setq trailer-offset (TeX-current-offset))
  			      (buffer-substring (point) (point-max))))))))))
+     ;; file name should be relative to master
+     (setq original (file-relative-name original (TeX-master-directory)))
      (save-excursion
        (set-buffer file-buffer)
        (setq buffer-undo-list t)
